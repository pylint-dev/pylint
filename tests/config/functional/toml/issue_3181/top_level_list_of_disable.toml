# This crashed previously see https://github.com/PyCQA/pylint/issues/3181
[tool.pylint]
max-line-length = 120
disable = ["C0121"]
