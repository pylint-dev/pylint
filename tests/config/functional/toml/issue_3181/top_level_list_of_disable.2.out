************* Module {abspath}
{relpath}:1:0: E0014: Out-of-place setting encountered in top level configuration-section 'max-line-length' : '120' (bad-configuration-section)
{relpath}:1:0: E0014: Out-of-place setting encountered in top level configuration-section 'disable' : '['C0330']' (bad-configuration-section)
